app/routes/admin.js

```diff
@@ -0,0 +1,69 @@
+const router = require("express").Router();
+const mongoose = require("mongoose");
+const Request = require("../../models/request");
+const User = require("../../models/users");
+const Donor = require("../../models/donor");
+const bodyParser = require("body-parser");
+
+const authCheck = (req, res, next) => {
+  if (!req.user) {
+    res.redirect("/auth/google");
+  } else {
+    next();
+  }
+};
+
+router.get("/", authCheck, (req, res) => {
+  console.log(req.user);
+  Request.find().then(currentRequest => {
+    console.log(currentRequest);
```

Comment:
currentRequest to currentRequests its kinda misleading

Reply:
Yeah.. I'll replace currentRequest with allRequests as this function is intended to return all the requests that are created.